doc/man7/provider-base.pod

```diff
@@ -108,6 +108,10 @@ provider):
  CRYPTO_secure_free             OSSL_FUNC_CRYPTO_SECURE_FREE
  CRYPTO_secure_clear_free       OSSL_FUNC_CRYPTO_SECURE_CLEAR_FREE
  CRYPTO_secure_allocated        OSSL_FUNC_CRYPTO_SECURE_ALLOCATED
+ BIO_new_file                   OSSL_FUNC_BIO_NEW_FILE
+ BIO_new_mem_buf                OSSL_FUNC_BIO_NEW_MEMBUF
+ BIO_read                       OSSL_FUNC_BIO_READ
+ BIO_free                       OSSL_FUNC_BIO_FREE
```

Comment:
I'm curious, what made you insert these lines here?

Reply:
These are passed to the fips provider..
It will use them for macing the fips module and the status indicator.  (once HMAC is usable)

Reply:
Sorry, I was vague...

I was just curious what made you place them between `CRYPTO_secure_allocated` and `OPENSSL_cleanse`?
(I noticed you order them differently in `crypto/provider_core.c` and in ` include/openssl/core_numbers.h`)

Do note that I'm not asking you to change the order, I'm just asking the question out of curiosity...

Reply:
I moved them up in one place in the code to be inside a #ifndef FIPS_MODE